packages/ui/src/Common.ts

```diff
@@ -698,6 +698,8 @@ export interface TextFieldProps extends FieldWithLabelsProps {
   inputRef?: any;
   onSubmitEditing?: any;
   onEnter?: any;
+  // if blurOnSubmit is false and multiline is true, return will create a new line
```

Comment:
Can you also mention that it defaults to true?

Reply:
done! :)